### DISCUSSION TITLE
How is this hatching like texture made?

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#517] [Q&A]
![image](https://github.com/bnpr/Malt/assets/36077492/e32ef3b9-e7f0-402e-8bb8-f875fe5dae49)
in https://github.com/bnpr/Malt/discussions/382

Beside, we have the camera uv in malt already. Do we have window uv also?
![image](https://github.com/bnpr/Malt/assets/36077492/85020e2e-5f49-4218-a707-08d765c3bba0)

**Comment by pragma37** [comment]
> Do we have window uv also?

Screen UV is the window uv.

If you mean an equivalent to the Camera Texture Coordinate, that would just be position in camera space.
![imagen](https://github.com/bnpr/Malt/assets/36610999/76fab376-9797-454c-b727-f1a32733778f)

Sorry for the late reply.
